Otec-iOS/Otec/LoginViewController.swift

```diff
@@ -13,63 +13,41 @@ class LoginViewController: UIViewController {
     @IBOutlet weak var newspaperTextfield: UITextField!
     override func viewDidLoad() {
         super.viewDidLoad()
-//        
-//        id = dictionary["id"],
-//        title = dictionary["title"],
-//        body = dictionary["body"],
-//        newspaper = dictionary["newspaper_name"],
-//        createdAt = News.creationDateFromDateString(dictionary["created_at"])
-//
-//        let newDictionary = ["id":"ABCDE12345",
-//                             "title":"Titulo",
-//                             "body":"Este es el contenido de la noticia. Ampliaremos.",
-//                             "newspaper_name":"Daily Planet",
-//                             "created_at":"19-12-1989"]
-//        
-//        guard let new = News.init(dictionary:newDictionary) else {
-//            print ("failure creating new")
-//            return
-//        }
-//        let future3 = NewsRepository().findAll()
-        
-        
         
-//        let future2 = NewsRepository().findAll()
-        
-        // Do any additional setup after loading the view.
+        self.checkIfLoggedInThenShowFeed()
     }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    private func checkIfLoggedInThenShowFeed() {
+        if (NSUserDefaults.standardUserDefaults().objectForKey(Constants.newspaperUserDefaultsKey) != nil) {
```

Comment:
I don't like a `UIViewController` hitting `NSUserDefaults` directly, like this.
Could we think of a `Session` object that encapsulates this, or something similar?